src/python/WMCore/MicroService/MSCore/MSAuth.py

```diff
@@ -89,21 +89,30 @@ def __init__(self, msConfig, **kwargs):
                 with open(authFile, 'rb') as istream:
                     self.authzKey = istream.read()
 
-    def authorizeApiAccess(self, service, action, method=None):
+    def userInfo(self):
         """
-        Check auth role.
-        :return: boolean
+        Read incoming HTTP request headers and return user info
+        :return: user structure
```

Comment:
it comes from `REST/Auth.py` `user_info_from_headers` function which provides`{'dn': None, 'method': None, 'login': None, 'name': None, 'roles': {}}` object. I don't see where it can be string. If you think so please clarify.

Reply:
From this line `                return "localhost-user"`, it will be a string.

Reply:
ok, got it, and fixed.